octobot_trading/personal_data/orders/order_factory.py

```diff
@@ -105,6 +106,8 @@ def create_order_instance(
         exchange_creation_params=exchange_creation_params,
         associated_entry_id=associated_entry_id
     )
+    if trigger_above is not None and isinstance(order, personal_data.LimitOrder):
+        order.trigger_above = trigger_above
```

Comment:
I don't think it's the proper place do to this. Why not providing it with order.update and then ignoring this data when it's not necessary?